fw-child/languages/cdc/fr_CA.po

```diff
@@ -1353,3 +1354,11 @@ msgstr "Sélectionnez pour afficher soit les valeurs de données absolues, soit
 #: template/map/controls/display.php:46
 msgid "View climate data as grid boxes, or aggregated by Census Subdivision, Health Region or Watershed."
 msgstr "Affichez les données climatiques sous forme de grilles ou regroupées par subdivision de recensement, région socio-sanitaire ou bassin versant."
+
+#: template/newsletter-hero.php:3
+msgid "Climatedata.ca Newsletter"
```

Comment:
Official website name is "ClimateData.ca" (with uppercase "D")

Reply:
No need for this as the `newsletter-hero.php` file has been disregarded.